receiver/elasticsearchreceiver/testdata/integration/Dockerfile.elasticsearch.7_0_0

```diff
@@ -1,4 +1,4 @@
-FROM elasticsearch:7.0.0
+FROM elasticsearch:7.17.9
 
```

Comment:
This one (and all the other elastic search files) should not be bumped (it is specifically testing the 7.0.0 version)

Reply:
Yes this should not be updated, looking at the description, renovatebot shouldn't be updating it, but i think this was left from the original PR.